src/structlog/processors.py

```diff
@@ -909,3 +911,75 @@ def __call__(
                 event_dict["event"] = replace_by
 
         return event_dict
+
+
+class CallsiteNamespaceAddr:
```

Comment:
ehm :)

```suggestion
class CallsiteNamespaceAdder:
```

it also needs to be added to the modules `__all__`